styles.css

```diff
@@ -20,6 +20,8 @@ a {
 
 .frontpage section {
   background-image: url(./Images/background/header-shapes\ mobile@2x.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
   margin: 0 3% 0 3%;
   padding-top: 15%;
 }
```

Comment:
- [ ] On the mobile screen the headline section needs to be correctly positioned as shown in the Figma design, a nice fix could be by exporting each piece as its own which is highlighted in red from the Figma design screenshot below and then set multiple background image to the headline section with positioning each decorative piece in its correct position, follow [CSS Multiple Backgrounds](https://www.w3schools.com/css/css3_backgrounds.asp) for more information 👏🏼 

| Your Design | Figma Design|
| -----------------  | ------------------- |
| ![image](https://user-images.githubusercontent.com/68030297/230241505-fc845348-e00b-45b3-b931-e57b5234fcd6.png) | ![image](https://user-images.githubusercontent.com/68030297/230241645-a09f8276-7fb8-4a2a-a62c-efb031313070.png) |